templates/company/supervise/index.html

```diff
@@ -107,20 +107,18 @@
 
 <script>
     google.charts.load('current', {'packages':['annotationchart']});
-    google.charts.setOnLoadCallback(drawChart);
-
-    function drawChart() {
-        var data = new google.visualization.DataTable();
-        data.addColumn('date', 'Date');
-        {% for h in employees_graph.header %}
-          data.addColumn('number', '{{h.1}}');
-        {% endfor %}
-
-        {% for d1 in employees_graph.data %}
-          data.addRow([new Date({{d1.0|sub:86400|mul:1000}}){% for d2 in d1.1 %}, {{d2.0}}{% endfor %}]);
-        {% endfor %}
-
-        var options = {
+    google.charts.setOnLoadCallback(()=>drawChart({{employees_graph}}));
```

Comment:
Not sure how well python dictionaries to JSON work

Reply:
Mmm it seems pretty broken to me... There is a syntax error.

Reply:
syntax error where? in the JS, or in the python?